### PR TITLE
#110-0rder_detail-view

### LINKED ISSUE #110: Create some view for order_details

### PR DESCRIPTION
**Pull request by NikolayVaradinov**
Create view and serializer with Django REST_FRAMEWORK

### CODE REVIEW

File: Carwash/Carwash/urls.py [hunk 1]
```diff
@@ -16,17 +16,9 @@
...
-router.register(r'users', UserViewSet)
-
-
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('car_washes.urls')),
     path('', include('users.urls')),
-]
+    path('', include('order_details.urls')),
+]
```
Comment (strangerok1507) [inline Carwash/Carwash/urls.py]: new lines after this list

File: Carwash/order_details/urls.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+from rest_framework import routers
+from order_details.views import OrderDetailViewSet
+
+router = routers.DefaultRouter()
+router.register(r'order_details', OrderDetailViewSet)
+
+
+urlpatterns = [
+    path('', include(router.urls)),
+    path('api-auth/', include('rest_framework.urls', namespace='rest_framework'))
```
Comment (strangerok1507) [inline Carwash/order_details/urls.py]: why is api-auth in the order_details? move it to root url

**Review by strangerok1507 — CHANGES_REQUESTED** [review]
looks good. need minor changes for approve

**Review by strangerok1507 — CHANGES_REQUESTED** [review]
need changes

File: Carwash/order_details/views.py [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+
+class OrderDetailViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows OrderDetail to be viewed or edited.
+    """
+    queryset = OrderDetail.objects.all().order_by('order')
+    serializer_class = OrderDetailSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def perform_create(self, serializer):
```
Comment (strangerok1507) [inline Carwash/order_details/views.py]: why do you add this method?

Comment (NikolayVaradinov) [reply Carwash/order_details/views.py]: Sorry, this is a mistake. It was prom REST_FRAMEWORK tutorial. I'll remove it.

**Comment by NikolayVaradinov** [comment]
So I suppose I should leave services for the time being.

**Review by strangerok1507 — APPROVED** [review]

**Comment by strangerok1507** [comment]
Looks good! +1

**Pull request merged**